addons/l10n_hu/models/account_move.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    @api.depends('country_code', 'move_type')
+    def _compute_show_delivery_date(self):
+        # EXTENDS 'account'
+        super()._compute_show_delivery_date()
+        for move in self:
+            if hasattr(move, 'country_code') and move.country_code == 'HU':
```

Comment:
Just want to say, when does the move not have the field country_code? It's a related, so we always have a value (even if False)

Reply:
I don't remember when and how but i had a traceback cause of the country_code that wasn't present on the move that's why i've added the check, now it may not be useful in 99% of the time but it can still be usefull as a protective thing i think ?

Reply:
I don't see how it's possible, but yeah, if it is useful sometimes, you should put it (but if not, then it's just useless computation)

Reply:
Maybe it was due to other thing i've done, i will check right now if i can reproduce the traceback :+1:

Reply:
I tend to agree with @gawa-odoo, I don't think `country_code` will ever be missing and I thought there was a more convincing reason to use `hasattr()`, but since it does not seem to be the case, i would rather remove it.
We can always fix it later if we realize there is an issue with it (but I doubt it).
We will avoid setting an example that might be replicated in the code later :wink:

Reply:
I just try it out, it doesn't crash anymore don't know why i need it back then, sorry for the time loss :sweat_smile:

Reply:
No worries :slightly_smiling_face: